jacqueline-chamnankool/wk01/2-tues/strings/strings.js

```diff
@@ -0,0 +1,129 @@
+// https://www.teaching-materials.org/javascript/exercises/strings
+
+/* DrEvil
+Create a function called DrEvil. It should take a single argument, an amount, 
+and return '<amount> dollars', except it will add '(pinky)' at the end if the 
+amount is 1 million. */
+function DrEvil(amount) {
+    let str = amount + " dollars";
+    if (amount == 1000000) {
+        str += " (pinky)";
+    }
+
+    return str;
+}
+
+/* MixUp
+Create a function called mixUp. It should take in two strings, and return 
+the concatenation of the two strings (separated by a space) slicing out and 
+swapping the first 2 characters of each. You can assume that the strings 
+are at least 2 characters long. */
+function mixUp(str1, str2) {
+    let tempChar = str1[0];
+    str1 = str1.replace(tempChar, str2[0]);
+    str2 = str2.replace(str2[0], tempChar);
+    //string.replace(character to replace, character to be replaced)
+    return str1 + " " + str2;
+}
+
+/* FixStart
+Create a function called fixStart. 
+It should take a single argument, a string, and return a version where all 
+occurences of its first character have been replaced with '*', except for the 
+first character itself. You can assume that the string is at least one 
+character long. */
+function fixStart(str) {
+    //need global g modifier
+    let firstChar = str[0];
+    //allows using variable in replace function
+    var reg = new RegExp(firstChar, "g");
```

Comment:
Kudos on incorporating regex in your solution! Just a reminder though that Joel recommends we steer clear of the `var` keyword unless it's specifically required.